### PR TITLE
Set icon background color for Windows 8 tiles

### PR DESCRIPTION
**Pull request by linquize**
Windows 8.1 automatically determines the background color, which is not always sensible.
The background is also orange, which gives no contrast to the main content.
win8gittile1

With this patch, a fixed background color is set. This makes the main content more clear.
win8gittile2

### CODE REVIEW

**Comment by linquize** [comment]
Yes, the installer works on Win 8.1 x64.

**Comment by dscho** [comment]
Thanks!

File: bin/sh.VisualElementsManifest.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<Application xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<VisualElements
+		BackgroundColor="#F0EFE7"
```
Comment (sschuberth) [inline bin/sh.VisualElementsManifest.xml]: Where does this color code come from, i.e. why is it not pure white (`#FFFFFF`)?

Comment (linquize) [inline bin/sh.VisualElementsManifest.xml]: git-scm.com background color

Comment (sschuberth) [inline bin/sh.VisualElementsManifest.xml]: Thanks, makes sense.

**Comment by sschuberth** [comment]
@linquize Could you please explain the steps you used to create your screenshots above? If I just search for "git" from the charms bar in Windows 8, my icons look quite fine even without your patch, see:
clipboard-1

**Comment by linquize** [comment]
On Win 8.1, I simply pressed the start button. Did not use search function.

**Comment by sschuberth** [comment]
Hmm, still, the background color looks fine here, i.e. it's transparent like for the other icons:
clipboard-1
Can it be that you have set your desktop color resolution to anything lower than 32-bit?

**Comment by linquize** [comment]
let me check my VM ...

**Comment by linquize** [comment]
It is 32-bit color depth.

**Comment by kusma** [comment]
git-icons

FWIW, it also looks strange here (before any patch is applied)

**Comment by sschuberth** [comment]
@kusma Are you using Windows 8 or 8.1 (in case that makes any difference)?

**Comment by kusma** [comment]
Windows 8.1

**Comment by linquize** [comment]
Before the patch, my Win 8.1 looks like @kusma 's Git Bash icon

**Comment by kusma** [comment]
@sschuberth Unfortunately, I don't have time to try out the patch this week.

**Comment by sschuberth** [comment]
@kusma That's fine. The patch in this thread has already been merged anyway. The pending one is in https://github.com/git-for-windows/sdk/pull/26. But I'll just merge that now as @linquize showed via screenshots that it works for him. As the patch is uncritical otherwise, that's enough proof for me right now.

**Pull request merged**